Assets/scripts/LoadCatalog.cs

```diff
@@ -259,6 +296,56 @@ private void GenerateCategories()
         Debug.Log($"Categories loaded: {result}");
     }
 
+    private void GenerateBrands()
+    {
+        foreach (Item item in loadedItems)
+        {
+            List<string> itemBrands = item.GetBrand();
+
+            foreach (string brand in itemBrands)
```

Comment:
Please move the second for each loop into another function to make it easier  to follow